src/main/java/com/google/sps/servlets/utils/CommentAnalysis.java

```diff
@@ -53,12 +53,26 @@ public CommentAnalysis(LanguageServiceClient languageService) {
    * @return a Statistics object that contains required values to display
    */
   public Statistics computeOverallStats(CommentThreadListResponse youtubeResponse) {
-    List<Double> scoreValues =
+    List<Sentiment> documentList =
```

Comment:
What is the documentList var holding?

Reply:
documentList holds all comments built by comment content retrieved from ThreadListResponse